# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1448339853 18000
#      Mon Nov 23 23:37:33 2015 -0500
# Node ID 554a39b4dd2dd2f534c75b1b6fa83e5f4ba5c164
# Parent  b6408331bfa2131566cb2342c386e43989fe02f4
NEWS: correct name of execution exception class

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -90,19 +90,19 @@ Summary of important user-visible change
       isequalwithequalnans    javamethods
       java_convert_matrix     re_read_readline_init_file
       java_debug              read_readline_init_file
       java_invoke             saving_history
 
  ** The global error_state variable in Octave's C++ API has been
     deprecated and will be removed in a future version.  Now the error
     and print_usage functions throw an exception
-    (octave_execution_error) after displaying the error message.  This
-    makes the error and print_usage functions in C++ work more like the
-    corresponding functions in the scripting language.
+    (octave_execution_exception) after displaying the error message.
+    This makes the error and print_usage functions in C++ work more like
+    the corresponding functions in the scripting language.
 
  ** New configure option, --enable-address-sanitizer-flags, to build
     Octave with memory allocator checks (similar to those provided by
     valgrind) built in.
 
 Summary of important user-visible changes for version 4.0:
 ---------------------------------------------------------
 
